app/(public)/page.tsx

```
@@ -1,19 +1,79 @@
-import { Resource } from '@prisma/client'
-import { SearchBox } from '@/components/search-box'
+// src/app/(public)/page.tsx
+import { ResourceStatus, Resource } from "@prisma/client";
+import { redirect } from "next/navigation";
+import { createClient } from "@/utils/supabase/server";
+import { Navbar } from "@/components/layout/navbar";
+import { SearchBox } from "@/components/search-box";
+import { getResources, getSuggestedSubjects } from "@/actions/resources";
+import { SubjectResources } from "@/components/subject-resources";
+
+async function handleGetUserData(supabase: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)